signer/src/storage/model.rs

```diff
@@ -257,8 +418,6 @@ pub struct SweptDepositRequest {
     /// The transaction ID of the bitcoin transaction that swept in the
     /// funds into the signers' UTXO.
     pub sweep_txid: BitcoinTxId,
-    /// The transaction sweeping in the deposit request UTXO.
-    pub sweep_tx: BitcoinTx,
```

Comment:
I couldn't see that this needed to be used anywhere so I removed it to avoid the extra I/O. But if it's needed later for validation etc. then we can easily add it back.